site-src/references/policy-attachment.md

```diff
@@ -219,45 +235,37 @@ the same resource _and_ have an identical field specified with different values,
 precedence MUST be determined in order of the following criteria, continuing on
 ties:
 
+* Hierarchical Policies override Direct Policies.
```

Comment:
I'd actually thought the opposite would be true 🙃. Ideally I think there would be no overlap in what you could configure in a hierarchical and direct policy though. Are there any scenarios where you'd want these kinds of policy attachment to interact with each other?

Reply:
Good point, I've updated.